File: Day 11/JatinSaini664.java
// package Day 11;
import java.util.*;

public class JatinSaini664 {
    public static int GCD(int a , int b){
        return (b!=0)?GCD(b, a%b):a;
    }
    public static void main(String[] args) {
        Scanner scn = new Scanner(System.in);

        int a =scn.nextInt();
        int b=scn.nextInt();

        int gcd=GCD(a, b);

        int lcm= (a*b)/gcd;

        System.out.println(lcm);
    }
}

File: Day 11/Jatin-Saini664.cpp
#include<bits/stdc++.h>

using namespace std;

int GCD(int a, int b){
    return (b!=0)?GCD(b, a%b):a;
}

int main(){
    int a, b;
    cin>>a>>b;
    int gcd=GCD(a, b);
    int lcm = (a*b)/gcd;
    cout<<lcm<<"\n";
}